tsdb/points_test.go

```diff
@@ -162,18 +162,24 @@ func TestParsePointMissingTagValue(t *testing.T) {
 func TestParsePointMissingFieldValue(t *testing.T) {
 	_, err := ParsePointsString(`cpu,host=serverA,region=us-west value=`)
 	if err == nil {
-		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, "cpu")
+		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, `cpu,host=serverA,region=us-west value=`)
 	}
 
 	_, err = ParsePointsString(`cpu,host=serverA,region=us-west value= 1000000000`)
 	if err == nil {
-		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, "cpu")
+		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, `cpu,host=serverA,region=us-west value= 1000000000`)
```

Comment:
Should this test case be split in two? Are there not two errors here, that need distinct testing? a) a field without a value, and b) a value without a field.